packages/common-lib/src/components/SunbirdPlayer.js

```diff
@@ -12,6 +12,12 @@ const SunbirdPlayer = ({
   ...props
 }) => {
   const { mimeType } = props
+
+  const typeMatch = mimeType.match(/\/(.+)$/)
+  const fileType = typeMatch ? typeMatch[1] : null
+
+  localStorage.setItem('contentType', fileType)
```

Comment:
@Aar-if Can we clear local storage once the use of this variable done?

Reply:
yes